test/DependencyInjection/AbtractServiceBusExtensionTestCase.php

```diff
@@ -0,0 +1,136 @@
+ * @copyright Copyright (c) 2016 prooph software GmbH (http://prooph-software.com/)
+ * @license   https://github.com/prooph/event-store-symfony-bundle/blob/master/LICENSE.md New BSD License
+ */
+
+declare (strict_types = 1);
+
+namespace ProophTest\Bundle\EventStore\DependencyInjection;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Prooph\Bundle\EventStore\DependencyInjection\ProophEventStoreExtension;
+use Prooph\EventStore\EventStore;
+use ProophTest\Bundle\EventStore\DependencyInjection\Fixture\Plugin\BlackHole;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\Compiler\ResolveDefinitionTemplatesPass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Dumper\Dumper;
+use Symfony\Component\DependencyInjection\Dumper\XmlDumper;
+use Symfony\Component\DependencyInjection\Dumper\YamlDumper;
+use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+use \Symfony\Component\DependencyInjection\ParameterBag\ParameterBag;
+
+abstract class AbtractServiceBusExtensionTestCase extends TestCase
+{
+    abstract protected function loadFromFile(ContainerBuilder $container, $file);
+
+    /**
+     * @test
+     */
+    public function it_creates_an_event_store()
+    {
+        $container = $this->loadContainer('event_store');
+
+        $config = $container->getDefinition('prooph_event_store.event_store.main_store');
+
+        self::assertEquals(EventStore::class, $config->getClass());
+
+        /* @var $eventBus EventStore */
+        $eventBus = $container->get('prooph_event_store.event_store.main_store');
+
+        self::assertInstanceOf(EventStore::class, $eventBus);
+    }
+
+    /**
+     * @test
+     */
+    public function it_creates_multiple_event_storees()
+    {
+        $container = $this->loadContainer('event_store_multiple');
+
+
+        foreach (['main', 'second'] as $name) {
+            $config = $container->getDefinition('prooph_event_store.event_store.' . $name . '_store');
+
+            self::assertEquals(EventStore::class, $config->getClass());
+
+            /* @var $eventBus EventStore */
+            $eventBus = $container->get('prooph_event_store.event_store.' . $name . '_store');
+
+            self::assertInstanceOf(EventStore::class, $eventBus);
+        }
+    }
+
+    /**
+     * @test
+     */
+    public function it_dumps_multiple_event_storees()
+    {
+        $this->dump('event_store_multiple');
+    }
+
+    private function loadContainer($fixture, CompilerPassInterface $compilerPass = null)
+    {
+        $container = $this->getContainer();
+
+        $container->registerExtension(new ProophEventStoreExtension());
+
+        $loadXml = new XmlFileLoader($container, new FileLocator(dirname(__DIR__).'/Resources/config'));
+        $loadXml->load('services.xml');
+
```

Comment:
Test services are not available in extension. How can I inject services for testing only?